install.sh

```diff
@@ -298,31 +317,6 @@ parse_command_line_arguments() {
         unset DO_UPDATE_ONLY
         INTERNAL_AUTOUPDATE="true"
     fi
-
-    # Using core.hooksPath implies it applies to all repo's
```

Comment:
This is no more needed -> `--single` is just a one repo install from now on!

Reply:
Ups that was wrong! Its needed! TEst 067.sh fails otherwise with tests-corehookspath.sh
But of course its needed....